### ISSUE TITLE
"Face_emotion_recognition" project is my Graduation project. The application of FER-model can recognize Facial-expression in real-time.

### ISSUE THREAD

**Issue by lyxwzx** [lyxwzx/Face_emotion_recognition#1]
The "Face_emotion_recognition" project include:

Using Tensorflow,keras... to train Fer2013 data, the training model (Accuracy-rate：66%) is saved as fer2013lyx66.h5.
Coding with Python and Opencv (Haar classifier), the Face-recognition Project can mark faces in real-time accurately and return the predicted Facial-emotion class and probability.
You can download the project, configure the environment，run with Pycharm.
